Backend/Underdogs-Struggling/pom.xml

```diff
@@ -17,6 +17,12 @@
 		<java.version>17</java.version>
 	</properties>
 	<dependencies>
+
+		<dependency>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+
```

Comment:
Please ensure that the indentation is consistent with other lines.

Reply:
Done